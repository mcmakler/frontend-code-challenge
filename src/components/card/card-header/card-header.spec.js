import React from 'react';
import CardHeader from './card-header';
import ReactTestRenderer from 'react-test-renderer';

describe('CardHeader specs', () => {
  it('should match snapshot', () => {
    const props = {
      thumb: '',
      tag: '',
      alt: ''
    };
    const component = ReactTestRenderer.create(<CardHeader {...props} />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
