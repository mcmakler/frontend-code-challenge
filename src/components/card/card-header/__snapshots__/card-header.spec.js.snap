// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader specs should match snapshot 1`] = `
<div
  className="cardHeader"
>
  <img
    alt=""
    className="cardHeader__img"
    src=""
  />
  
</div>
`;
